src/server/pkg/deploy/cmds/cmds.go

```diff
@@ -218,7 +220,8 @@ func DeployCmd(noMetrics *bool) *cobra.Command {
 	deploy.PersistentFlags().StringVar(&etcdVolume, "static-etcd-volume", "", "Deploy etcd as a ReplicationController with one pod.  The pod uses the given persistent volume.")
 	deploy.PersistentFlags().BoolVar(&dryRun, "dry-run", false, "Don't actually deploy pachyderm to Kubernetes, instead just print the manifest.")
 	deploy.PersistentFlags().StringVar(&logLevel, "log-level", "info", "The level of log messages to print options are, from least to most verbose: \"error\", \"info\", \"debug\".")
-	deploy.PersistentFlags().BoolVar(&enableDash, "dash", false, "If true, deploy the Pachyderm UI along with Pachyderm (experimental)")
+	deploy.PersistentFlags().BoolVar(&enableDash, "dashboard", false, "If true, deploy the Pachyderm UI along with Pachyderm (experimental)")
+	deploy.PersistentFlags().BoolVar(&dashOnly, "dashboard-only", false, "If true, only deploy the Pachyderm UI (experimental), without the rest of pachyderm. This is for launching the UI adjacent to an existing Pachyderm cluster")
```

Comment:
I think you could remove the `If true` part of the help message here, people will assume that the description applies to the case where the value is `true`. Also because users are unlikely to type `--dashboard=true` instead of `--dashboard`, although I think that both will work with this flag library.